docs/configuration.rst

```diff
@@ -1,6 +1,15 @@
 Configuration
 #############
 
+Configuration files for *gallery-dl* use a JSON-based file format.
+
+For a (more or less) complete example with options set to their default values, see gallery-dl.conf_.
+
+For a configuration file example with more involved settings and options, see gallery-dl-example.conf_.
```

Comment:
Any reason you didn't just copy-paste the lines from README, including the `| ` at the front of every line here?

Reply:
Er, I guess I just couldn't remember the meaning of | at the start of the line; but I added a blank line in between, so the resulting effect is the same, isn't it?

Reply:
Not exactly (*), but its fine. I was just curious.
Thanks for your contributions; all of them!

(*)
blank lines produce a new paragraph (`<p>`), while lines starting with `|` will be shown as if they only had a manual line break between them (`<br/>`)